src/main/java/com/tenx/ms/retail/orders/rest/dto/Order.java

```diff
@@ -0,0 +1,78 @@
+package com.tenx.ms.retail.orders.rest.dto;
+
+import com.tenx.ms.commons.validation.constraints.Email;
+import com.tenx.ms.commons.validation.constraints.PhoneNumber;
+import com.tenx.ms.retail.common.util.DenyConverterAccess;
+import com.tenx.ms.retail.orders.domain.OrderStatus;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.Date;
+import java.util.List;
+
+
+@ApiModel("Order Model")
+public class Order {
+    @ApiModelProperty(value = "The order Id.", readOnly = true)
+    private Long orderId;
+    @ApiModelProperty(value = "The store Id.", readOnly = true)
```

Comment:
it would be better to have a separate empty line between each variable. it would be much clean & easy to read